app/src/main/res/layout/fragment_match_list.xml

```diff
@@ -6,15 +6,10 @@
     android:layout_height="match_parent"
     tools:context=".ui.matchlist.MatchListFragment">
 
-    <TextView
-        android:id="@+id/text_gallery"
+    <ListView
+        android:id="@+id/matchListView"
```

Comment:
I think we are using naming conventions like.... list_view_match_list (except for a few places we forgot to make it consistent). Can we update the id to follow the same naming convention?